src/Common/Auth/AuthUtils.php

```diff
@@ -106,6 +106,11 @@ public function __construct($mode = '')
                 privStatement("UPDATE `globals` SET `gl_value` = ? WHERE `gl_name` = 'hidden_auth_dummy_hash'", [$this->dummyHash]);
             }
         }
+        if ($GLOBALS['password_expiration_days'] === '') {
```

Comment:
like the premise here (set it to 0 when it is blank).
To ensure security (in case this class is ever loaded with globals not correctly loaded (maybe in a bad module or something like that where it would revert a openemr instance to a less secure setting; ie. change the password expiration setting to a set value to 0) would consider something like this:
```
        if ($GLOBALS['password_expiration_days'] === '') {
            $sqlQuery = sqlQuery("SELECT 'gl_value' FROM `globals` WHERE `gl_name` = 'password_expiration_days'");
            if ($sqlQuery['gl_value'] === '') {
                $GLOBALS['password_expiration_days'] = 0;
                sqlQuery("UPDATE `globals` SET `gl_value` = '0' WHERE `globals`.`gl_name` = 'password_expiration_days' AND `globals`.`gl_index` = '0'");
                error_log("Blank global password_expiration_days updated to 0");
            } else {
                error_log("OpenEMR ERROR: global password_expiration_days was not defined");
                die();            
        }
```